daemon/migrations/20220404113202_create_closed_cfds_table.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS closed_cfd_kind (
+    id integer PRIMARY KEY autoincrement,
+    uuid text UNIQUE NOT NULL,
+    kind text NOT NULL
+);
+CREATE TABLE IF NOT EXISTS closed_cfds (
+    id integer PRIMARY KEY autoincrement,
+    uuid text UNIQUE NOT NULL,
+    position text NOT NULL,
+    initial_price text NOT NULL,
+    taker_leverage integer NOT NULL,
+    settlement_interval integer NOT NULL,
+    n_contracts integer NOT NULL,
+    counterparty_network_identity text NOT NULL,
+    role text NOT NULL,
+    opening_fee integer NOT NULL,
+    initial_funding_rate text NOT NULL,
+    initial_tx_fee_rate text NOT NULL,
+    fees integer NOT NULL,
+    lock_txid text NOT NULL,
+    lock_dlc_vout text NOT NULL,
+    -- optional closing price. Applies to collaborative settlement and
+    -- CET, but not to refund
+    closing_price text,
+    -- optional collaborative settlement tx data
+    close_txid text,
+    close_payout text,
+    close_vout text,
+    -- optional commit tx data
+    commit_txid text,
+    -- optional cet data
+    cet_txid text,
+    cet_vout text,
+    cet_payout text,
+    -- optional refund tx data
+    refund_txid text,
+    refund_vout text,
+    refund_payout text
+);
```

Comment:
Hmm, can you explain this design a bit more?

I would have expected something like this:

- `transactions` table with columns `id`, `closed_cfd_id`, `kind`, `transaction_id`, `vout`, `value`
- `closed_cfds` table
- `transactions` would hold entries for `lock`, `close`, `cet`, `commit` and `refund`

You can then `JOIN` together the representation that you currently have in `closed_cfds` (if you wanted to) or also just make a query that only returns you refunded CFDs.

Reply:
> Hmm, can you explain this design a bit more?

I opted for something very simple, but I assumed that we could do something better :D I will follow your advice here.

Reply:
Let me know if the new design makes more sense!